src/gcp_scanner/crawl.py

```diff
@@ -368,18 +369,29 @@ def get_bucket_names(project_name: str, credentials: Credentials,
       break
 
     for bucket in response.get("items", []):
-      buckets_dict[bucket["name"]] = (bucket, None)
+      buckets_dict[bucket["name"]] = (bucket, [])
+
+      if dump_fd is None:
```

Comment:
Like I said in the previous comment. You already have everything including root folder name. All you need is to just add that data. No need for extra request. Take a look at what API returns you. Also could you please make sure to test your code before submitting PRs? I see extra null field in the response output.

Reply:
Ok, working on it